Checkout/Checkout/Source/Validation/Validators/CardValidator.swift

```diff
@@ -83,6 +83,15 @@ public class CardValidator: CardValidating {
     return cardNumberValidator.validate(cardNumber: cardNumber)
   }
 
+  /// Checks whether a given card number is at least a partial match for any of the supported schemes.
+  /// - Parameters:
+  ///   - cardNumber: The card number to validate.
+  /// - Returns: The card's scheme if successful, else an error.
+  public func eagerValidate(cardNumber: String) -> Result<Card.Scheme, ValidationError.CardNumber> {
+    logManager.queue(event: .validateCardNumber)
```

Comment:
I hope our logger has a large enough buffer to handle logging on each user input.

Reply:
@harry-brown-cko is there required to log each time we call eagerValidate ? , This will be called on every character changed in text field right ?

Reply:
Log is only called once, check the log manager